C++/CountPrimes.cpp

```diff
@@ -0,0 +1,41 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+int countPrimes(int n){
+    bool p[n+1] = {false};
+        //p[0] = false, p[1] = false;
+        for(int i=2;i<n;i++){
```

Comment:
instead of line no. 7 for loop you can initialize p[n+1] = {true} then 
p[0] = false; 
p[1] = false; 
so we can save the unnecessary iteration.